end-to-end-tests/src/helpers/ctx.rs

```diff
@@ -54,24 +64,16 @@ impl Context {
     }
 }
 
-pub fn build_client() -> Result<oxide_client::Client> {
-    let mut headers = HeaderMap::new();
-    headers.insert(
-        AUTHORIZATION,
-        HeaderValue::from_static(
-            "Bearer oxide-spoof-001de000-05e4-4000-8000-000000004007",
-        ),
-    );
+pub async fn build_client() -> Result<oxide_client::Client> {
+    build_authenticated_client().await
+}
 
-    let client = reqwest::ClientBuilder::new()
-        .default_headers(headers)
-        .connect_timeout(Duration::from_secs(15))
-        .timeout(Duration::from_secs(60))
-        .build()?;
-    Ok(Client::new_with_client(&get_base_url(), client))
+fn rss_config() -> Result<SetupServiceConfig> {
+    toml::from_str(RSS_CONFIG_STR)
+        .with_context(|| format!("parsing {:?} as TOML", RSS_CONFIG_PATH))
 }
 
-pub fn nexus_addr() -> SocketAddr {
+pub async fn nexus_addr() -> SocketAddr {
```

Comment:
Could make this an `anyhow::Result<SocketAddr>` to replace all the `unwrap`'s w/ ?

Reply:
Done in the latest commits.